nextflow.config

```diff
@@ -29,5 +29,9 @@ profiles {
         
     }
 
+    docker {
+	docker.enabled = true
+    }
```

Comment:
In the main branch we have Docker enabled by default and example has a `nodocker` profile. I would keep this approach, b.c. basically we don't document how to install all depencencies. The README.md is incorrect, though, there indeed is no `docker` profile..